.env

```
@@ -1,128 +1,2 @@
-
-RECREATE_QA_USERS=false
-
-###> rostering file-upload ###
-FILESYSTEM_ROSTERING_FILES_ADAPTER_S3_PREFIX=
-ALLOWED_UPLOADED_FILE_MAX_SIZE=5242880
-ALLOWED_UPLOADED_FILE_MAX_RECORDS=10000
-ROSTERING_EXTERNAL_REPORTING_SYSTEM_ENABLED=false
-ROSTERING_EXTERNAL_REPORTING_SYSTEM_BASE_URL=
-###< file-upload ###
+APP_ENV=docker
+COMPOSER_HOME='~/.config/composer'
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)